docker/generate_build_matrix.py

```diff
@@ -32,7 +32,7 @@ def _get_pytorch_version(python_version: str):
 def _get_torchvision_version(pytorch_version: str):
     if pytorch_version == '1.10.2':
         return '0.11.3'
-    if pytorch_version == '1.11.0':
+    if pytorch_version == '1.12.0':
         return '0.12.0'
```

Comment:
My guess is we need 0.13.0 for torch 1.12?
```suggestion
        return '0.13.0'
```

Reply:
Yes, torchvision should be upgraded to 0.13.0